src/main/java/com/eternalcode/discordapp/leveling/games/CodeGameAnswerController.java

```diff
@@ -66,7 +66,7 @@ public void onMessageReceived(MessageReceivedEvent event) {
                 .setDescription(formatter.format(this.codeGameConfiguration.embedSettings.description))
                 .setFooter(this.codeGameConfiguration.embedSettings.footer);
 
-            this.experienceService.modifyPoints(event.getAuthor().getIdLong(), points, true)
+            this.experienceService.modifyPoints(event.getAuthor().getIdLong(), points, true, event.getChannel().getIdLong())
```

Comment:
From what I see, `event.getChannel()` is already used twice in this class
Store a variable.